### PR TITLE
Refactor routes

### PR DESCRIPTION
**Pull request by damymetzke**
As you'll see I really like splitting routes into more files.
I've commented what is being added to each file.

### CODE REVIEW

File: routes/admin.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Route::get('/', AdminDashboardController::class)->name('index');
+
+Route::prefix("books")->name("books.")->group(function (){
+    Route::get('/', [AdminBookController::class, 'index'])->name('index');
+    Route::get('create', [AdminBookController::class, 'create'])->name('create');
+    Route::post('/', [AdminBookController::class, 'store'])->name('store');
+    Route::get('{book}/edit', [AdminBookController::class, 'edit'])->name('edit');
+    Route::put('{book}', [AdminBookController::class, 'update'])->name('update');
+    Route::delete('{book}', [AdminBookController::class, 'destroy'])->name('destroy');
```
Comment (krekas) [inline routes/admin.php]: Using resource here would delete couple lines. If want you can set `except('show')`

File: routes/admin.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Route::put('approve/{book}', [AdminBookController::class, 'approveBook'])->name('approve');
+});
+
+Route::prefix("users")->name("users.")->group(function (){
+    Route::get('/', [AdminUsersController::class, 'index'])->name('index');
+    Route::get('{user}/edit', [AdminUsersController::class, 'edit'])->name('edit');
+    Route::put('{user}', [AdminUsersController::class, 'update'])->name('update');
+    Route::delete('{user}', [AdminUsersController::class, 'destroy'])->name('destroy');
```
Comment (krekas) [inline routes/admin.php]: Same here why not use resource?
You will be able to compare your solution with the one that Povilas will post in a separate video, next week.

Comment (damymetzke) [reply routes/admin.php]: I think you might be looking at the wrong commit.
The latest commit (d140299) has both `Route::resource` calls you are suggesting.

Comment (krekas) [reply routes/admin.php]: You are right. But it's very hard to read your commits when every line changed from double " to single ' quotes

**Pull request closed without merging**